pkg/storage/internalstorage/config.go

```diff
@@ -35,7 +35,10 @@ type Config struct {
 
 	Params map[string]string `yaml:"params"`
 
-	Log *LogConfig `yaml:"log"`
+	Log        *LogConfig `yaml:"log"`
+	MaxBackups int        `yaml:"maxBackups"`
```

Comment:
These new parameters are all log parameters. They should be placed in the log so that users can configure them.